crates/amaru-ledger/src/rules/transaction.rs

```diff
@@ -92,7 +92,7 @@ pub fn execute(
     is_valid: bool,
     transaction_body: KeepRaw<'_, MintedTransactionBody<'_>>,
     transaction_witness_set: &MintedWitnessSet<'_>,
-    transaction_auxiliary_data: Option<&AuxiliaryData>,
+    transaction_auxiliary_data: Option<Hash<32>>,
```

Comment:
NITPICK: this is a bit confusing, probably should be named `transaction_auxiliary_data_hash` or something.

Reply:
I specifically meant the argument name, but also yes there should be an alias.